api/v1alpha1/conditions_const.go

```diff
@@ -20,11 +20,18 @@ import clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 
 const (
 	// DeviceReadyCondition reports on current status of the device. Ready indicates the device is in a Running state.
+	// Equivalent to ContainerProvisionedCondition of CAPD.
```

Comment:
Please remove hints to CAPD. This is evident for everybody interested in the topic if they compare the two provider integrations.

Reply:
fixed.